docs/chainlink-keepers/cron-contracts.md

```diff
@@ -0,0 +1,629 @@
+
+You will also need a MetaMask (or other Remix compatible wallet) in your browser with some testnet tokens of choice. See our [MetaMask tutorial](https://www.youtube.com/watch?v=4ZgFijd02Jo) if you need help setting up a wallet.
+
+> ⚠️ Testing Contracts
+> We recommend you first test the tutorial on an EVM-compatible testnet before moving to mainnet.
+
+## Creating a Job Scheduler contract
+
+To deploy an on-chain Job Scheduler contract, we will use the externally audited and pre-deployed Chainlink  [CronUpkeepFactory](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/factories/CronUpkeepFactory.sol) smart contract. This is called the [Factory pattern](https://www.youtube.com/watch?v=Q1zZo4O_Ong), where one contract deploys another contract. To deploy your Job Scheduler (`CronUpkeep`), select the new file icon in Remix and name it `CronUpkeepFactory.abi`. Paste the [ABI](#cronupkeepfactory-abi) for `CronUpkeepFactory` from the [Resources](#resources) section below. The ABI is the application binary interface that is used to interact with the smart contract that has been deployed to the network.
+
+![Remix ABI Cron](/images/keepers/cron-1.png)
+
+Select the **DEPLOY & RUN TRANSACTIONS** icon in the left-hand navigation bar and set **Environment** to *Injected Web3*. Paste the address of our deployed `CronUpkeepFactory` contract in the **At Address** box and select the **At Address** button. You can find the [addresses](#contract-addresses) for deployed `CronUpkeepFactory` contracts in the [Resources](#resources) section. You will then be able to view callable functions of `CronUpkeepFactory`. Select the `newCronUpkeep` button and follow the prompts in MetaMask to confirm the transaction. When the transaction is complete,  a transaction hash will appear in the terminal. This points to the creation of your Job Scheduler.
+
+## Finding the address of your Job Scheduler
+
+Copy the transaction hash and open it in the scanner of the chain you are working on. Once you have found it, select the **Internal Txns** tab. This displays the contract creation and contract address in the `To` field. Note the address and transaction hash for future use.
+
+![Remix Transaction Hash](/images/keepers/cron-2.png)
+
+## Interacting with your Job Scheduler
+
+To interact with your Job Scheduler contract, create a new file called `CronUpkeep.abi` in Remix and paste the [ABI](#job-scheduler-aka-cronupkeep-abi) for `CronUpkeep` contract from the [Resources](#resources) section below. Select the **DEPLOY & RUN TRANSACTIONS** icon on the left and paste the address of your Job Scheduler contract in the **At Address** box, and then select the **At Address** button. Accept the prompts after which the contract interface will open in the bottom left of Remix.
```

Comment:
```
To interact with your Job Scheduler contract,
```

Clarify that it's an ABI and not a contract. I went back in the docs looking to see which steps I missed because I thought there was a contract I didn't deploy.

Also clarify that when users deploy `CronUpkeep.abi`, they should deploy at the address for their `CronUpkeepFactory.abi` address in the Remix Deployed Contracts section.

Reply:
Done.